templates/broker/broker.conf.erb

```diff
@@ -191,3 +191,10 @@ DEFAULT_REGION_NAME="<%= scope.lookupvar('::openshift_origin::conf_broker_defaul
 # Allow region selection when creating applications. Default is true
 ALLOW_REGION_SELECTION="<%= scope.lookupvar('::openshift_origin::conf_broker_allow_region_selection') %>"
 
+# Create new gear UUIDs (and thus gear usernames) with the format:
+#    <domain_namespace>-<app_name>-<gear_index>
+USE_PREDICTABLE_GEAR_UUIDS="<%= scope.lookupvar('::openshift_origin::conf_broker_use_predictable_gear_uuids') %>"
+# Gear UUIDs that are too large to fit in 32 char limit will fall back to 24-hex chars.
+# Reject app creation if domain + app name larger than this:
+# (set to 26 to leave 2 chars for hyphens and 4 chars for up to 9999 gears.)
+LIMIT_APP_NAME_CHARS=26
```

Comment:
@IshentRas Should LIMIT_APP_NAME_CHARS be conditioned on ::openshift_origin::conf_broker_use_predictable_gear_uuids? If so perhaps wrap this block in an if statement.

@sosiouxme mind verifying this is all that's necessary to produce a sane predictable gear uuid config?